tools/topology/topology2/include/components/eqiir.conf

```diff
@@ -61,4 +61,5 @@ Class.Widget."eqiir" {
 	no_pm			"true"
 	num_input_pins		1
 	num_output_pins		1
+	no_wname_in_kcontrol_name	"true"
```

Comment:
why not set this to true in widget-common.conf?

Reply:
ack, seems that base class is best place.

Reply:
done